builtin/logical/transit/path_sign_verify.go

```diff
@@ -63,6 +78,12 @@ to the min_encryption_version configured on the key.`,
 				Type:        framework.TypeBool,
 				Description: `Set to 'true' when the input is already hashed. If the key type is 'rsa-2048' or 'rsa-4096', then the algorithm used to hash the input should be indicated by the 'algorithm' parameter.`,
 			},
+			"signature_algorithm": &framework.FieldSchema{
+				Type:    framework.TypeString,
+				Default: "pss",
```

Comment:
I don't think we should have a default, it ties this strongly to RSA. Instead I'd suggest the default behavior is just dependent on the type of key and this stays a blank string.